packages/@o3r/stylelint-plugin/package.json

```diff
@@ -26,9 +27,16 @@
     "tslib": "^2.5.3"
   },
   "peerDependencies": {
+    "@schematics/angular": "~17.2.0",
     "postcss": "^8.4.5",
+    "postcss-scss": "^4.0.9",
```

Comment:
I could not find where this dep is used . Is it something we have forgotten to add before ?

Reply:
I originally added it because I wanted the ng-add to install it, this might not be needed
then again, it doesn't really make sense to use this plugin without `postcss-scss` in the stylelint config